lib/outcome.rb

```diff
@@ -0,0 +1,18 @@
+class Outcome
+
+  def is_tie?(all_move_combos, full_board_bool)
+    full_board_bool && !winner?(all_move_combos) ? true : false
+  end
+
+  def winner?(all_move_combos)
```

Comment:
This should be `has_winner?` since the return type will be a true or false.

Reply:
done!